Tests/ContentKeyCrudTests.cs

```diff
@@ -142,18 +143,42 @@ public void AddContentKey_WithVariousInvalidData_Fails()
 		}
 
 		[Fact]
-		public void Save_WithSneakilyCorruptedContentKey_Fails()
```

Comment:
Perhaps useful to keep this test, i.e. "save after sneaky corruption". But since null is now valid, make the key invalid some other way. E.g. use a length that is not supported.

Reply:
Hmm, maybe the code comparator is confusing me here. It seems this test still exists, but is now named "Save_WithInvalidContentKey_Fails()"? Perhaps keep the original naming as it's used elsewhere as well I think?

Reply:
I think the "sneaky" part in such tests implies that invalid data was created by modifying a value directly, instead of adding/removing actual entities. So maybe useful to keep that in title?